include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_v2r3.hpp

```diff
@@ -126,7 +127,11 @@ struct GridwiseGemm_k0mk1_k0nk1_mn_xdlops_v2r3
 
     using ThisThreadBlock = ThisThreadBlock<BlockSize>;
 
+#if 0
```

Comment:
Maybe we still need V1 pipeline as default, because it can apply 2-wave's strategy.